tests/foreman/sys/test_hot_backup.py

```diff
@@ -0,0 +1,228 @@
+# -*- encoding: utf-8 -*-
+"""Test class for ``katello-backup``
+
+@Requirement: HotBackup
+
+@CaseAutomation: notautomated
+
+@CaseLevel: System
+
+@CaseComponent: Backup
+
+@TestType: System
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+from robottelo.decorators import stubbed, tier3
+from robottelo.test import TestCase
+
+
+class HotBackupTestCase(TestCase):
+    """Implements ``katello-backup --online`` tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(HotBackupTestCase, cls).setUpClass()
+
+    @tier3()
+    @stubbed()
+    def test_positive_directory_exists(self):
+        """katello-backup with existing directory
+
+        @id: 3f8285d5-c68b-4d84-8568-bcd7e4f7f19e
+
+        @Setup:
+
+        Populate a Satellite with hosts/repos/contents
+
+        @Steps:
+
+        1. Create a directory (ie /tmp/backup1)
+        2. Run ``katello-backup /tmp/backup1``
+        3. Restore from backup
+
+        @Assert: The backup and restore succeed.
+
+        @CaseAutomation: notautomated
+
+        @CaseLevel: System
+        """
+        pass
```

Comment:
`pass` here is not needed at all. python classes accepts only docstring in its body.

```python
class Foo(object):
    """This class does bar
    """
```

is accepted, `pass` is not needed